gameScreen.html

```diff
@@ -31,18 +31,19 @@
             <div class="sidePanel">
                 <p>Username's stats</p>
                 <br>
-                <p>Max HP : 55</p>
-                <p>Level : 10</p>
-                <p>Xp : 102/124</p>
-                <p class="specialLine"><span>Coins : 47  </span><img src="/assets/UI/traderMap.png" alt=""></p>
+                <p id="MaxHp">Max HP : </p>
+                <p id="Level">Level : 10</p>
+                <p id="Xp">Xp : 102/124</p>
+                <p class="specialLine"><span id="Coins">Coins : 47  </span><img src="/assets/UI/traderMap.png" alt=""></p>
                 <br>
-                <p class="specialLine"><button>+</button><span>Strength : 8</span></p>
-                <p class="specialLine"><button>+</button><span>Endurance : 11</span></p>
+                <p class="specialLine"><button id="strength_Button">+</button><span id="Strength">Strength : </span></p>
```

Comment:
Let's have consistent id name:
```suggestion
                <p class="specialLine"><button id="StrengthButton">+</button><span id="Strength">Strength : </span></p>
```